flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/ExecutionGraph.java

```diff
@@ -1028,8 +1029,12 @@ public void registerExecutionListener(ActorGateway listener) {
 	
 	private void notifyJobStatusChange(JobStatus newState, Throwable error) {
 		if (jobStatusListenerActors.size() > 0) {
+			SerializedThrowable st = null;
```

Comment:
`st` is not really descriptive. A slightly longer variable name could be helpful.